python/README.rst

```diff
@@ -8,15 +8,19 @@ has been slightly modified to be of use to a broader community.
 Current State
 -------------
 
-In its current state, the wrapper only provides orbit propagation
-functionality - ranging and other orbit fitting capabilities will be
-added in the near future.
+In its current state, the wrapper provides functionality for
+transformations between orbital element definitions, the computation
+of ephemerides, and orbit propagation - ranging and other orbit
+fitting capabilities will be added in the near future.
 
-Documentation will be added in the near future, too. For now, we refer
-to the ``test.py`` file for an example usecase, as well as `Lynne
-Jones' Jupyter notebook
+Some documentation and installation guides can be found below - more
+documentation will be added in the near future. Please refer to the
+``test.py`` file for some example usecases, as well as `Lynne Jones'
+Jupyter notebook
 <https://github.com/rhiannonlynne/notebooks/blob/master/PyOorb%20Demo.ipynb>`_.
```

Comment:
Would it be useful to create an "Examples" directory here, to add this demo notebook? I can create a PR for that if so.

Reply:
I tried updating your demo notebook to make it work with the changes that I applied to pyoorb, but I am having issues using pyoorb within jupyter: from some reason `oorb_init` is not able to read `OBSCODE.dat` although `OORB_DATA` is set properly. I am not having these issues when running the same code in the terminal. Unless you have a quick idea what might be the issue, I will defer my work on the notebook and commit what I currently have with a note that it doesn't seem to work properly.

Reply:
Is it related to SIP? Is the environment variable access/set inside the notebook?

Reply:
Also I'm confused -- OBSCODE.dat or the ephemeris file (DE403.dat, etc) is the problem?

Reply:
It's OBSCODE.dat and the error occurs within the FORTRAN code (error code 1). Within the notebook I can read the variable value for `OORB_DATA`...

Reply:
Let me have a look at this later today - I didn't have any problems previously, but there are certainly various reasons why it could be problematic. 
Given that you don't have a problem in the console, but you do in Jupyter, it seems likely it's a path issue .. are you using OSX or linux to run the notebook?

Reply:
linux: ubuntu 16.04; thanks!